Tiltfile

```diff
@@ -356,6 +356,12 @@ def strip_sec_ctx(yaml):
             spec["securityContext"] = {}
             for container in spec.get("containers", []):
                 container["securityContext"] = {}
+        if data.get("kind") == "Namespace":
+            spec = data["metadata"]["labels"]
+            pod_security_standard_label =  "pod-security.kubernetes.io/enforce" 
+            spec.pop(pod_security_standard_label)
```

Comment:
This raises `KeyError` if label is not found. Add default value of `None` to avoid it.